pkg/client/results/reader.go

```diff
@@ -111,18 +118,30 @@ func DiscoverVersion(reader io.Reader) (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "error extracting config")
 	}
-	var version string
+
+	parsedVersion, err := validateVersion(conf.Version)
```

Comment:
I'll take a look; I originally just copied the code snippet from another location and I think the validation may have done another task there which I removed, so it may not be necessary now.